admin/run-python3-tests

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3.3
+#!/usr/bin/env python3
```

Comment:
Yeah that accidentally got pushed, I wasn't sure if it's something you'd want or not.  I'll take it out. :)